tox.ini

```diff
@@ -1,11 +1,22 @@
 [tox]
 # We do not currently test with pypy because psycopg2 does not work there.
-envlist = py27,py37
+envlist =
+  py27
+  py37
+  lint
 
 [testenv]
-# Run a single test with e.g. tox -e py37 test/sqlalchemy/test_run_transaction.py
+deps =
+  -r test-requirements.txt
 commands =
-     py.test -p sqlalchemy.testing.plugin.pytestplugin {posargs:test/sqlalchemy}
-deps = --requirement=test-requirements.txt
-# For some reason pip fails to load the requirements file without this.
-setenv = LANG=en_US.utf-8
```

Comment:
Is there any reason to believe this is no longer necessary? I can't remember whether it mattered in CI or when run locally. I think it might be better to carry this forward into the new version.

Reply:
I actually have no clue if this is needed. Everything is running fine on CI and locally (linux), but to be on the safe side I will re-add it (macOS might behave differently and I'm not able to test this).